docs/content/en/docs/reference/clusterspec/vsphere.md

```diff
@@ -18,55 +18,55 @@ The following additional optional configuration can also be included:
             - 10.96.0.0/12
-   controlPlaneConfiguration:
-      count: 1
-      endpoint:
+   controlPlaneConfiguration:        <a href="#controlplaneconfiguration-required"># Specific cluster control plane config (required)</a>
+      count: 1                       <a href="#controlplaneconfigurationcount-required"># Number of control plane nodes (required)</a>
+      endpoint:                      <a href="#controlplaneconfigurationendpointhost-required"># IP for control plane endpoint (required)</a>
          host: ""
-      machineGroupRef:
+      machineGroupRef:               <a href="#controlplaneconfigurationmachinegroupref-required"># vSphere-specific Kubernetes node config (required)</a>
         kind: VSphereMachineConfig
         name: my-cluster-machines
-      taints:
+      taints:                        <a href="#controlplaneconfigurationtaints"># Taints applied to control plane nodes </a>
       - key: ""
         value: ""
         effect: ""
-      labels:
+      labels:                        <a href="#controlplaneconfigurationlabels"># Labels applied to control plane nodes </a>
         "<key1>": ""
         "<key2>": "" 
-   datacenterRef:
-      kind: VSphereDatacenterConfig
+   datacenterRef:                    <a href="#datacenterref"># Kubernetes object with vSphere-specific config </a>
+      kind: VSphereDatacenterConfig  <a href="#kubernetesversion-required"># Version of Kubernetes to use for cluster (required)</a>
       name: my-cluster-datacenter
    externalEtcdConfiguration:
-     count: 3
-     machineGroupRef:
+     count: 3                        <a href="#externaletcdconfigurationcount"># Number of etcd members </a>
+     machineGroupRef:                <a href="#externaletcdconfigurationmachinegroupref"># vSphere-specific Kubernetes etcd config</a>
         kind: VSphereMachineConfig
         name: my-cluster-machines
-   kubernetesVersion: "1.25"
-   workerNodeGroupConfigurations:
-   - count: 1
-     machineGroupRef:
+   kubernetesVersion: "1.25"         <a href="#kubernetesversion-required"># Kubernetes version to use for the cluster (required)</a>
+   workerNodeGroupConfigurations:    <a href="#workernodegroupconfigurations-required"># List of node groups you can define for workers (required) </a>
+   - count: 1                        <a href="#workernodegroupconfigurationscount"># Number of worker nodes </a>
+     machineGroupRef:                <a href="#workernodegroupconfigurationsmachinegroupref-required"># vSphere-specific Kubernetes node objects (required) </a>
        kind: VSphereMachineConfig
        name: my-cluster-machines
-     name: md-0
-     taints:
-     - key: ""
+     name: md-0                      <a href="#workernodegroupconfigurationsname-required"># Name of the worker nodegroup (required) </a>
+     taints:                         <a href="#workernodegroupconfigurationsname-required"># Taints to apply to worker node group nodes </a>
```

Comment:
```suggestion
     taints:                         <a href="#workernodegroupconfigurationstaints"># Taints to apply to worker node group nodes </a>
```

Reply:
Another great catch. I knew I was going to mess up at least one of those.

Reply:
Fixed.